src/stories/dragToCreate/DragToCreate.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Timeline from '../../timeline';
+import { d, someHumanResources, someTasks } from '../sampleData';
+import { ComponentStory } from '@storybook/react';
+import { dragToCreateScenarios } from './DragToCreateScenarios';
+import { DragToCreateParam } from '../../types';
+
+export default {
+    title: 'Features/Drag to create',
+    excludeStories: ["DragToCreateDemo"],
+    component: Timeline
+};
+
+export const Main: ComponentStory<typeof Timeline> = () => {
+    return <DragToCreateDemo />;
```

Comment:
In tab-ul `Story` se vede doar `return <DragToCreateDemo />`, cred ca ar trebui sa se vada si implementarea lui `DragToCreateDemo` pentru ca utilizatorii ar trebui sa vada exact cum se foloseste functionalitatea; cred ca poti sa bagi clasa in Main. Astfel poti sa stergi si `excludeStories` din default.
![image](https://user-images.githubusercontent.com/68424941/228907466-d32d54be-2263-4397-8733-512449b0b7a2.png)
Mai e si un rand gol mai jos.